danger/Dangerfile.swift

```diff
@@ -0,0 +1,7 @@
+import Danger
+import Foundation
+
+let danger = Danger()
+
+SwiftLint.lint(.modifiedAndCreatedFiles(directory: "Sources"), inline: true)
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/c5a4410df1234342937e628f80ec930f9115845a/danger/Dangerfile.swift#L7"></span>Files should have a single trailing newline.<br />
<code>trailing_newline</code> <code>Dangerfile.swift:7</code></del></td>
    </tr>
  </tbody>
</table>